common-test/build.gradle

```diff
@@ -13,6 +13,7 @@ android {
 }
 
 dependencies {
+    api project(":core")
```

Comment:
interesting why does it need to be `api` ?

Reply:
I think it can be implementation. But we are using all dependencies as `api` in `commontest` module, so I didn't change that. But yes, we can improve this in future.